sympy/series/approximants.py

```diff
@@ -53,7 +58,9 @@ def approximants(l, X=Symbol('x'), simplify=False):
     ========
 
     sympy.concrete.guess.guess_generating_function_rational
-    mpmath.pade
+    :py:obj:`!mpmath.pade`
```

Comment:
I'm not sure what the syntax is for intersphinx references and I'm also not sure that mpmath.pade is the correct reference for the function.

I suggest just deleting this mpmath reference.